src/app/contextBroker/contextBroker.cpp

```diff
@@ -271,6 +274,7 @@ bool            statNotifQueue;
 #define PIDPATH             _i "/tmp/contextBroker.pid"
 #define IP_ALL              _i "0.0.0.0"
 #define LOCALHOST           _i "localhost"
+#define MAX_PERIOD          (3600 * 24 * 31)  // Once a month ...
```

Comment:
MAX_SUMMARY_TRACE_PERIOD ...
That is just TOO long.
It is only used in the parseargs vector, and only once.

What I could do is to define ONE_MONTH and use it.
Yes, I like that idea